tasks-desktops.yml

```diff
@@ -1,4 +1,6 @@
 ---
+- import_role:
+    name: lnls-ans-role-repositories
```

Comment:
Sorry for being picky. =] But doesn't this change belong in another PR?

Reply:
you are right. "given enough eyeballs all bugs and lazy commits are shallow", I heard. done.